code/modules/jobs/job_types/roguetown/garrison/dungeoneer.dm

```diff
@@ -54,11 +54,14 @@
 		H.mind.adjust_skillrank(/datum/skill/craft/cooking, 1, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/misc/sewing, 1, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/craft/traps, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/reading, 1, TRUE)
```

Comment:
they can finally read the laws they're meant to uphold!!

Reply:
Comprehending the tome of justice is important.